time/time.go

```diff
@@ -7,3 +7,10 @@ func ElapsedTime(f func()) time.Duration {
 	f()
 	return time.Now().Sub(startTime)
 }
+
+func GivenTimeIsBetweenTimeRange(givenTime, time1, time2 time.Time) bool {
```

Comment:
Can we rename the function something like `IsTimeBetween`

Reply:
changed like this `IsTimeBetweenRange`